tests/test_immutabledict.py

```diff
@@ -160,3 +162,20 @@ def test_immutabledict_duplication_blocking(self):
         immutabledict_from_unique_keys(good)
         immutabledict((x for x in good), forbid_duplicate_keys=True)
         immutabledict_from_unique_keys(x for x in good)
+
+
+def test_inverse():
+    output = immutabledict({"foo": "bar", "bar": "bat"})
+    output = output.inverse()
+
+    reference = immutabledict({"bar": "foo", "bat": "bar"})
+    assert output == reference
```

Comment:
Two things:
1. Let's simplify this by comparing the dicts directly, rather than assigning them to variables first.
2. Use `unittest`'s `assertEqual`